pom.xml

```diff
@@ -1664,23 +1665,24 @@
       </activation>
     </profile>
 
-    <!-- spark3 is an alias of spark3.2 -->
+    <!-- spark3 is an alias of spark3.3 -->
     <profile>
       <id>spark3</id>
       <properties>
-        <spark3.version>3.2.1</spark3.version>
+        <spark3.version>3.3.0</spark3.version>
         <spark.version>${spark3.version}</spark.version>
-        <sparkbundle.version>3</sparkbundle.version>
+        <sparkbundle.version>3.3</sparkbundle.version>
```

Comment:
Shouldn't `sparkbundle.version` still be `3` for this profilie? After building code with -Dspark3 option, I see the bundle named `hudi-spark3.3-bundle-*` instead of `hudi-spark3-bundle-*`. Is that expected?